flipperdevices/flipperzero-firmware
# [FL-3769] Check universal remote files before loading

# What's new

- Verify that the universal remote files before allowing the user to start brute forcing.
- Slightly refactor affected code.

# Caveats
- Due to the files being checked for errors, the load times are longer.

# Verification 

1. Introduce errors into one of remote library files (or get an invalid one from #3399)
2. Try loading the universal remote that has errors
3. Observe the error
4. Repeat step 2, but with a valid file
5. No error should be shown

# Checklist (For Reviewer)
